internal/clientconn/listener.go

```diff
@@ -111,22 +113,22 @@ func Listen(opts *NewListenerOpts) (*Listener, error) {
 		}
 
 		close(l.unixListenerReady)
-		ll.Info(fmt.Sprintf("Listening on Unix %s...", l.UnixAddr()))
+		ll.InfoContext(ctx, fmt.Sprintf("Listening on Unix %s...", l.UnixAddr()))
 	}
 
 	if l.TLS != "" {
 		var config *tls.Config
 
 		if config, err = tlsutil.Config(l.TLSCertFile, l.TLSKeyFile, l.TLSCAFile); err != nil {
-			return nil, err
+			return nil, lazyerrors.Error(err)
```

Comment:
That's one of the few places where the error string is user-visible. Try to start FerretDB with a non-existing TLS file and see the output. It should be nice, and lazyerrors will make it worse.

Let's add a comment so we don't forget.

Reply:
Indeed, lazyerror is not pretty. Added a comment